test/README

```diff
@@ -3,33 +3,49 @@
 This directory contains automated integration tests for Cockpit, and the support
 files for them.
 
-To run the tests on Fedora, you need to install the following packages:
-
-    $ sudo yum install npm qemu-kvm python \
-         firewalld libvirt-daemon libvirt-client libvirt-python \
-         openssl expect rsync curl xz
+To run the tests on Fedora, refer to the [HACKING](../HACKING.md) guide for
+installation of all of the necessary build and test dependencies. There's
+no need to trigger a build manually - the test suite preparation step below
+will handle that.
 
 You may need to reboot after installing the above packages. For initialization
 the following command needs to be run as root to setup the libvirt bridged network:
 
     $ sudo ./test/vm-prep
 
+Note that this step also attempts to ensure nested virtualization is enabled,
+and may emit a spurious `rmmod: ERROR: Module kvm_amd is not currently loaded`
+warning when running on an Intel CPU (or vice-versa).
```

Comment:
Perhaps something like this:

    [ -d /sys/modules/kvm_intel ] && rmmod kvm-intel && modprobe kvm-intel || true

and same for amd? that should avoid the noise and still show relevant errors.

This is aside from the fact that these modprobe.d snippets don't actually seem to work -- I need to boot with `kvm-intel.nested=1` on the kernel command line to really enable nested kvm.